geometry_msgs/Quaternion orientation
geometry_msgs/Twist twist
geometry_msgs/Vector3 acc
geometry_msgs/Vector3 acc_calib
geometry_msgs/Vector3 acc_angle
geometry_msgs/Vector3 mag
float32 mag_heading
float32 height
float32 dheight
float32 height_reference
float32 dheight_reference
